packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -15,8 +15,6 @@ import 'text_button.dart';
 import 'text_button_theme.dart';
 import 'theme.dart';
 
-const double _singleLineVerticalPadding = 14.0;
```

Comment:
We would probably want to keep this constant so we know it is the default rather than just a number in the constructor.

Reply:
@Piinks  Understood :bow:

Reply:
Committed: 739f429